app/assets/stylesheets/sorry.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the sorry controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
I know this was auto-generated, but would you mind removing it since there is no actual content?